config/importmap.rb

```diff
@@ -8,3 +8,4 @@
 pin '@hotwired/stimulus-loading', to: 'stimulus-loading.js', preload: true
 pin_all_from 'app/javascript/controllers', under: 'controllers'
 pin '@rails/ujs', to: 'https://ga.jspm.io/npm:@rails/ujs@7.0.3-1/lib/assets/compiled/rails-ujs.js'
+pin_all_from "app/javascript/custom", under: "custom"
```

Comment:
Do you know how this file exactly works?

Reply:
Yes, like it is importing or loading files which we have mentioned in it.